components/camel-grok/src/test/resources/org/apache/camel/component/grok/data/access_log

```diff
@@ -0,0 +1,1536 @@
+64.242.88.10 - - [07/Mar/2004:16:05:49 -0800] "GET /twiki/bin/edit/Main/Double_bounce_sender?topicparent=Main.ConfigurationVariables HTTP/1.1" 401 12846
```

Comment:
I guess this is a file to be ignored to check-in

Reply:
This file is used in GrokFileUnmarshalTest. But it was quite large, I have trimmed it to more reasonable 5 lines.